### PR TITLE
Updated ecs file with count to enable container insights and variables file with value.

### PR DESCRIPTION
**Pull request by kbodige**
Updated ecs file with count to enable container insights and variables file with value.

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -77,10 +77,9 @@ variable "ec2_ingress_sg_id" {
...
 
 //----------------------------------------------------------------------
-// ECS Cluster Variables
+// ECS Cluster Variables -- And setup a toggle (true/false)
 //----------------------------------------------------------------------
 variable "container_insights_enablement" {
   description = "Whether container sights are set, valid values are [enabled,disabled]"
-  type        = string
-  default     = "disabled"
+  container_insights_enablement     = 1
```
Comment (sienkin) [inline variables.tf]: This is an invalid `variable` declaration. The argument `container_insights_enablement` has no place here.

You've also removed the `type` and `default` arguments.

**Comment by sienkin** [comment]
I'm unsure as to what this PR is achieving.

The module already has optional support for enabling ContainerInsights that can be toggled based on the provided value of the `container_insights_enablement` variable. This means a value of `enabled` only needs to be passed in from the codebase that is using this module.

https://github.com/companieshouse/developer-site-stack

**Comment by kbodige** [comment]
Removed Count value in "ecs.tf" & Updated "variables.tf" with type & default arguments.

**Comment by marcransome** [comment]
Superseded by #10 (and release [1.1.4](https://github.com/companieshouse/terraform-library-ecs-cluster/releases/tag/1.1.4)).

**Pull request closed without merging**